src/protocols/secure_channel/CASESession.h

```diff
@@ -283,6 +288,8 @@ class DLL_EXPORT CASESession : public Messaging::UnsolicitedMessageHandler,
     // Sigma1 initiator random, maintained to be reused post-Sigma1, such as when generating Sigma2 S2RK key
     uint8_t mInitiatorRandom[kSigmaParamRandomNumberSize];
 
+    int mSequence = 0;
```

Comment:
This needs to be documented, and also documented why we increment it in CASESession.cpp

Reply:
I added some comments.